src/NHSD.BuyingCatalogue.Ordering.Api/Extensions/OrderExtensions.cs

```diff
@@ -57,10 +58,13 @@ public static bool IsAssociatedServicesSectionComplete(this Order order)
             return !(order is null) && order.AssociatedServicesViewed;
         }
 
-        public static bool IsSectionStatusComplete(this Order order, int serviceRecipientsCount, int catalogueSolutionsCount, int associatedServicesCount)
+        public static bool IsSectionStatusComplete(this Order order, int serviceRecipientsCount)
         {
             if (!IsFundingSourceComplete(order)) return false;
 
+            int catalogueSolutionsCount = order.OrderItems.Count(y => y.CatalogueItemType.Equals(CatalogueItemType.Solution));
+            int associatedServicesCount = order.OrderItems.Count(y => y.CatalogueItemType.Equals(CatalogueItemType.AssociatedService));
```

Comment:
Anything better than `y` for the lambda param? (I think `o` or `i` would be better.)

Reply:
sure